packages/taskrunner/README.md

```diff
@@ -0,0 +1,11 @@
+# Terarium Task Runner
```

Comment:
I wasn't sure where else to put this comment, so here will do :)

We'll need to update the `hmiServerDev.sh` to start this up locally

Reply:
I've added the container to the `docker-compose-common.yml` and that should really be enough to kick it off locally.

Getting a local dev container that hotswaps both the taskrunner code and the GoLLM code will be a `nice to have` that I'll get in after this PR.

Reply:
👍